src/main.js

```diff
@@ -1,14 +1,37 @@
 'use strict';
 
 const API_KEY = 'e9979803c3894fa182b399c482279297';
-const HOST_URL = 'https://newsapi.org/v2';
-const SOURCES_PARAM = '/top-headlines';
-
+const HOST_URL = 'https://newsapi.org/v2/';
+const SOURCES_PARAM = 'top-headlines';
 const url = `${HOST_URL}${SOURCES_PARAM}?country=us&apiKey=${API_KEY}`;
 const req = new Request(url);
+const container = document.getElementById('container');
+const fragment = document.createDocumentFragment();
+const getRandomIntNumber = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
+const getRandomImageUrl = () => `./src/img/news-${getRandomIntNumber(1, 3)}.jpg`;
+const loader = document.getElementById("loader");
+
+window.onload = () => loader.style.display = "none";
```

Comment:
1. Why do you hide loader with on `window.onload` event? By default it should be visible and should be hidden when data from server is populated. `window.onload` event in any way is not connected with data loading.

2. No reason to modify element props directly. It's better to manipulate classes. For example, you can make a class `hidden` class and add\remove this class as a reaction for different events.

Reply:
The loader was reworked taking into consideration these comments.